.semaphore/semaphore.yml

```diff
@@ -331,28 +324,6 @@ blocks:
     secrets:
     - name: google-service-account-for-gce
 
-- name: "Felix: Static checks on e1-standard-8"
```

Comment:
I don't think we need to run this in its own block and use up another standard-8 instance - should work fine on a standard-4 inline with building the code.